content/program-security.md

```diff
@@ -1,14 +1,8 @@
 # Create a Basic Program, Part 3 - Basic Security and Validation
 
-### List links for further reading here:
-
-- [Neodyme Security Workshop](https://workshop.neodyme.io/)
-- [Solana Smart Contracts: Common Pitfalls](https://blog.neodyme.io/posts/solana_common_pitfalls/)
-- [How to Audit a Solana Smart Contract](https://medium.com/coinmonks/how-to-audit-solana-smart-contracts-part-1-a-systematic-approach-56a434f6c9ed)
-
 # Lesson Objectives
 
-*By the end of this lesson, you will be able to:*
+_By the end of this lesson, you will be able to:_
```

Comment:
@ZYJLiu not a big deal, but all the other lessons use the single asterisk for italics.  Mind setting it back?

Reply:
@ZYJLiu I see there are a couple other italics in the "Think like an attacker" section that you updated to underbars... Mind setting those back as well? (I don't see any other instances in the project which use underbars for italics, errywhere else is asteriks `*`)

Thanks!

Reply:
Probably just happened from the editor he's using or whatever. I can update all of those pretty easily while I'm going through and doing my edits